hs_core/views/utils.py

```diff
@@ -800,6 +847,13 @@ def irods_path_is_allowed(path):
         raise SuspiciousFileOperation("File paths cannot contain '/./'")
 
 
+def irods_path_is_directory(istorage, path):
+    """ return True if the path is a directory. """
+    folder, base = os.path.split(path)
+    listing = istorage.listdir(folder)
+    return base in listing[0]
```

Comment:
There is already a ```is_folder()``` you wrote that does the same thing as this function, so this function is not needed.